suite/tests/CMakeLists.txt

```diff
@@ -4271,9 +4280,11 @@ if (BUILD_CLIENTS)
         set(cmd_pfx "sudo@")
         set(${testname_full}_precmd
           "foreach@${cmd_pfx}${CMAKE_COMMAND}@-E@remove_directory@${testname_full}.*.dir")
```

Comment:
Another test we should add is to run basic counts on the user+kernel trace, to count the number of kernel instrs. It'd need some changes to basic counts too.

Reply:
I will add a end marker first.